user/reference/xenial.md

```diff
@@ -0,0 +1,153 @@
+---
+title: The Xenial Build Environment
+layout: en
+---
+
+## What This Guide Covers
+
+On this page, you can find what is included on the Xenial build environment.
+
+## Using Xenial
+
+To get your builds using Ubuntu Xenial 16.04, add the following to your `.travis.yml`:
+
+```yaml
+dist: xenial
+```
+{: data-file=".travis.yml"}
+
+Please note that Xenial is available on our fully virtualized
+infrastructure. If you are running an Enterprise installation, please reach out
+to [enterprise@travis-ci.com](mailto:entereprise@travis-ci.com) to see how you can use the Xenial Docker images.
+
+## Image updates from Trusty
+
+Xenial includes the following changes and improvements:
+
+### Third party apt-repositories removed
+
+While third party apt-repositories are used during the Xenial image provisioning, they are all removed from the Xenial build image. This has two benefits; a) reduced risk of unrelated interference and b) faster apt-get updates.
+
+To specify a third party apt-repository, you can [add the source with the apt addon](/user/installing-dependencies/#adding-apt-sources) and specify the packages. For example:
```

Comment:
Do you need to add both? I think not. You could add just the source? Or the source and the package? We could maybe head off some confusion here.